packages/nest/src/generators/application/lib/update-tsconfig.ts

```diff
@@ -9,6 +9,16 @@ export function updateTsConfig(tree: Tree, options: NormalizedOptions): void {
     (json) => {
       json.compilerOptions.emitDecoratorMetadata = true;
       json.compilerOptions.target = 'es2015';
+      if (options.strict) {
+        json.compilerOptions = {
+          ...json.compilerOptions,
```

Comment:
Can you also apply the same settings to `@nx/nest:lib` generator? That was called out as different in the issue.